A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_IdentPage.js

```diff
@@ -25,17 +92,17 @@ class CDUIdentPage {
         mcdu.setTemplate([
             ["A320-200"],
             ["\xa0ENG"],
-            ["LEAP 1A-26[color]green"],
+            ["LEAP-1A26[color]green"],
             ["\xa0ACTIVE NAV DATA BASE"],
-            ["\xa0" + (date.length === 13 ? date[3] + date[4] + date[0] + date[1] + date[2] + "-" + date[8] + date[9] + date[5] + date[6] + date[7] : date) + "[color]cyan", "AIRAC[color]green"],
+            ["\xa0" + calculateActiveDate(date) + "[color]cyan", "AIRAC[color]green"],
             ["\xa0SECOND NAV DATA BASE"],
-            ["{small}{04MAY-04JUL{end}[color]cyan"],
-            ["", "STORED"],
-            ["", "{green}10{end}{small}RTES\xa0{end}{green}10{end}{small}RWYS{end}"],
-            ["CHG CODE", "[b-text]{big}{green}20{end}{end}WPTS\xa0{big}{green}20{end}{end}NAVS"],
-            ["{small}[  ]{end}[color]cyan", "DELETE ALL}[color]cyan"],
+            ["{small}{" + calculateSecDate(date) + "{end}[color]inop"],
+            ["", ""],//["", "STORED"]
+            ["", ""],//["{green}10{end}{small}RTES\xa0{end}{green}10{end}{small}RWYS{end}"] Orginal code
+            ["CHG CODE", ""],//["CHG CODE", "[b-text]{big}{green}20{end}{end}WPTS\xa0{big}{green}20{end}{end}NAVS"] Orginal code
+            ["{small}[  ]{end}[color]inop", ""], //["{small}[  ]{end}[color]cyan", "DELETE ALL}[color]cyan"], Orginal code
```

Comment:
Don't commit these, just remove them. They can always be found in the Git history.

Reply:
You want me to remove the inop from the brackets

Reply:
No, I'd like you to remove the comments, everything after // (including the //).

Reply:
ok will do